jmix-data/data/src/main/java/io/jmix/data/impl/dbms/HsqlSequenceSupport.java

```diff
@@ -51,7 +51,7 @@ public String getNextValueSql(String sequenceName) {
 
     @Override
     public String getCurrentValueSql(String sequenceName) {
-        return "select START_WITH from INFORMATION_SCHEMA.SYSTEM_SEQUENCES where SEQUENCE_NAME = '"
+        return "select (cast(next_value as bigint) - 1) from INFORMATION_SCHEMA.SYSTEM_SEQUENCES where SEQUENCE_NAME = '"
```

Comment:
@ilushaaa Won't this query increase the current sequence value? I mean if I call the getCurrentValue() multiple times, then we will get a different value for each invocation, right?

Were there problems with the `CURRENT VALUE FOR {SEQUENCENAME}` approach?